contentcuration/contentcuration/utils/assessment/qti/html/display.py

```
@@ -0,0 +1,36 @@
+from pydantic import Field
+
+from contentcuration.utils.assessment.qti.base import TextType
+from contentcuration.utils.assessment.qti.html.base import FlowContentElement
+from contentcuration.utils.assessment.qti.html.base import HTMLElement
+from contentcuration.utils.assessment.qti.html.content_types import FlowContent
+from contentcuration.utils.assessment.qti.html.content_types import FlowContentList
+from contentcuration.utils.assessment.qti.html.text import BlockHTMLText
+from contentcuration.utils.assessment.qti.html.text import InlineHTMLText
+
+
+class Label(InlineHTMLText):
```

Pylint (3.3.7)

[refactor] Too many ancestors (8/7)

(R0901)